third_party/move/move-stdlib/src/natives/debug.rs

```diff
@@ -180,7 +180,7 @@ mod testing {
     ) -> PartialVMResult<MoveStructLayout> {
         let annotated_type_layout = context.type_to_fully_annotated_layout(ty)?;
         match annotated_type_layout {
```

Comment:
let's replace this match with if let Some(MoveTypeLayout::Struct(..)) = to avoid fallthrough _

Reply:
done